### DISCUSSION TITLE
Request: Merge Duplicate Resources

### DISCUSSION THREAD

**Discussion by bpepple** [Metron-Project/metron#169] [General]
If you come across duplicate entries in the database you can make a request to merge them here. Just add a message in this discussion with the following message:

- **Type**: Character, Creator, Arc, Team
- **Canonical ID**: The Metron ID for the correct entry
- **Other ID**: The Metron ID for the duplicate entry

**Comment by pickled-dev** [comment]
- Type: Series
- Canonical ID: 8555
- Metron ID: 8556

**Reply by bpepple** [reply]
Deleted the second one, since there was no data to merge.

**Comment by pickled-dev** [comment]
Type: Series
Canonical ID: 2430
Other ID: 8613

**Reply by bpepple** [reply]
Deleted duplicate series.

**Comment by pickled-dev** [comment]
Type: Series
Canonical ID: 469
Other ID: 8686

**Reply by bpepple** [reply]
Duplicate series deleted.

**Comment by pickled-dev** [comment]
Type: Creator
Canonical ID: 972
Other ID: 11979

**Reply by bpepple** [reply]
Done.

**Comment by frijolie** [comment]
Type: Creator
Canonical ID: 8517
Other ID: 13187

**Reply by bpepple** [reply]
Done.

**Comment by frijolie** [comment]
Type: Creator
Canonical ID: 12452
Other ID: 13013

**Reply by bpepple** [reply]
I ended up deleted the bogus entries for D Wassel & A Wassel, since they couldn't be merged since both creators had multiple entries for individual issues.

**Comment by warhorus** [comment]
Type: Creator
Canonical ID: 8853
Other ID: 16868

**Reply by bpepple** [reply]
Done.
